src/Utils/General/DOMHelper.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace LearnosityQti\Utils;
```

Comment:
This doesn't match the directory path of the class file, which includes the subspace `General`. You're going to have issues with the autoloader, as this is not PSR-4 compatible.

Reply:
Done